pydhi/helpers.py

```diff
@@ -0,0 +1,14 @@
+def safe_length(input_list):
+    """
+    Get the length of a Python or C# list.
+
```

Comment:
You could consider using some well-defined docstring format, [either Google or Numpy](https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html#google-vs-numpy).

Reply:
yes, I agree, would be nice to change the formatting of docstrings to one of those. I chose to follow the existing style in this repo (had a look at docstrings for other functions in the same file).